platform/pulpcore/app/apps.py

```diff
@@ -17,6 +17,23 @@ def pulp_plugin_configs():
         if isinstance(app_config, PulpPluginAppConfig):
             yield app_config
 
+def pulp_plugin_config(plugin_app_label):
```

Comment:
Could we rename this? `get_plugin_config` would be my vote. The object names in this module already feel a little overloaded.